src/tpm2-header.c

```diff
@@ -4,7 +4,6 @@
  * All rights reserved.
  */
 #include <assert.h>
-#include <endian.h>
```

Comment:
Why remove `endian.h` here?  This file uses `be16toh` so we need `endian.h` no? It may be pulled in by another include but I'd prefer to include it explicitly.

Reply:
There are three files that use be<N>toh: functions src/tpm2-command.c, src/tpm2-header.c, and src/tpm2-response.c. Only tpm2-header.c included it explicitly. The others pulled it via <tss2/tss2_mu.h>. I just made it consistent.

Reply:
thx for the details